crates/sozo/ops/src/migration/mod.rs

```diff
@@ -174,3 +180,87 @@ enum ContractDeploymentOutput {
 enum ContractUpgradeOutput {
     Output(UpgradeOutput),
 }
+
+pub fn generate_overlays(ws: &Workspace<'_>) -> Result<()> {
+    let profile_name =
+        ws.current_profile().expect("Scarb profile expected to be defined.").to_string();
+
+    // its path to a file so `parent` should never return `None`
+    let manifest_dir = ws.manifest_path().parent().unwrap().to_path_buf();
+    let profile_dir = manifest_dir.join(MANIFESTS_DIR).join(profile_name);
+
+    let base_manifests = profile_dir.join(BASE_DIR);
+
+    let world = OverlayClass { name: WORLD_CONTRACT_NAME.into(), original_class_hash: None };
+    let base = OverlayClass { name: BASE_CONTRACT_NAME.into(), original_class_hash: None };
+
+    // generate default OverlayManifest from base manifests
+    let contracts = overlay_dojo_contracts_from_path(&base_manifests.join(CONTRACTS_DIR))
+        .with_context(|| "Failed to build default DojoContract Overlays from path.")?;
+    let models = overlay_model_from_path(&base_manifests.join(MODELS_DIR))
+        .with_context(|| "Failed to build default DojoModel Overlays from path.")?;
+
+    let default_overlay =
+        OverlayManifest { world: Some(world), base: Some(base), contracts, models };
+
+    let overlay_path = profile_dir.join(OVERLAYS_DIR);
```

Comment:
The folder may not exist, which causes the generation to fail. We should ensure the directory exists / is created when the command is run.